src/base.js

```diff
@@ -24,8 +24,9 @@ module.exports = {
     "unicorn/numeric-separators-style": "off",
     "unicorn/no-await-expression-member": "off",
     "@typescript-eslint/ban-ts-comment": "off",
+    "unicorn/import-style": "off",
     "unicorn/prevent-abbreviations": [
-      "error",
+      "warn",
```

Comment:
would turn it off completely. I think it should be the developers responibility to choose proper variable names and this hould also be taken care of in code reviews from the team if needed.